clients-simple/pom.xml

```diff
@@ -16,17 +16,22 @@
   -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
     <parent>
         <groupId>uk.gov.gchq.palisade</groupId>
         <artifactId>clients</artifactId>
         <version>0.4.0-${revision}</version>
         <relativePath>../pom.xml</relativePath>
     </parent>
 
-    <!--  *********** Aritfact ID ***********    -->
+    <modelVersion>4.0.0</modelVersion>
```

Comment:
Why has this tag been moved? Can this be moved back to where it was.

Reply:
Done

Reply:
The tag I was talking about was `<modelVersion>`. I would prefer this to be below the main `<project>` tag as this is where it appears in the other pom files.
The badly spelt `<!--  *********** Aritfact ID ***********    -->` comment can be removed.

Reply:
Ok,  I was interested in grouping the modelVersion, artifactId and the version tags together.  I have put all three at the top.